exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
+
+Lucian's girlfriend is on her way home and he hasn't cooked their anniversary dinner!
+
+In this exercise, you're going to write some code to help Lucian cook an exquisite lasagna from his favorite cookbook.
+
+You have four tasks, all related to the time spent cooking the lasagna.
+
+## 1. Define the expected oven time in minutes
+
+Define the `ovenTime()` method that does not take any parameters and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
+
+```cpp
+ovenTime();
```

Comment:
it is important, that when you do the definition the `;` is not needed!
the distinction between declaration and definition is not clear for many people and the error messages are not helpful in this case.

Reply:
Yes, that can be tricky. This is something we should definitely stress in the concept about functions.